shared/src/main/java/alcatraz/shared/LobbyKey.java

```diff
@@ -0,0 +1,12 @@
+
+package alcatraz.shared;
+
+public class LobbyKey {
+    public final long id;
```

Comment:
What for is the id of the key? Should not the secret already be unique, thus identifies the key 🤔

Reply:
was renamed to lobbyId to make it clearer